### PR TITLE
Fixed margin error on pages that use public_layout - issue  #347

### PR DESCRIPTION
**Pull request by cleisonsilfer**
Added div "pad-wrapper" for margin correction on pages.
Relative issue  #347

### CODE REVIEW

File: core/templates/public_layout.html [hunk 1]
```diff
@@ -44,25 +44,27 @@
...
-            </div>
-          </div>
-        {% endif %}
-
-        {% bootstrap_messages messages %}
-
-        {% block content %}{% endblock content %}
-      </div>
-    </div>
+    <div id="pad-wrapper">
```
Comment (luanfonceca) [inline core/templates/public_layout.html]: You need to fix this indentations. I recommend you to Spaces instead of Tabs on your text editor.

Comment (luanfonceca) [reply core/templates/public_layout.html]: And that `pad-wrapper` class, I think that you need to commit a `.css` file, right? Or it is a class from Bootstrap?

Comment (luanfonceca) [reply core/templates/public_layout.html]: Another thing, can you add a printscreen from the "Signup" and "Events" pages? For to look how its changing the layout.

Comment (cleisonsilfer) [reply core/templates/public_layout.html]: I fixed the indentation.
The css class already exists in the project, there was no need for inclusion.
See the result:
print1
print2

**Comment by cleisonsilfer** [comment]
I fixed the indentation.
The css class already exists in the project, there was no need for inclusion.
See the result:
print1
print2

**Comment by luanfonceca** [comment]
Can you show me how the Events page are?
As i remember, in there we didnt need to add the `pad-wrapper`.

**Comment by cleisonsilfer** [comment]
The Event page uses a different template and was not affected by my change. This page already had the "pad-wrapper" and served as an example for me.
print3

**Comment by luanfonceca** [comment]
Awesome!

**Comment by luanfonceca** [comment]
When you fix that indentation problem, i merge it.

**Comment by cleisonsilfer** [comment]
The problem continues? I made a commit by swapping Tab for spaces. Still need some fix?

**Comment by luanfonceca** [comment]
Oh ok, i haven't seen it

o/

**Comment by luanfonceca** [comment]
screen shot 2017-10-19 at 19 35 57


Done!

**Comment by cleisonsilfer** [comment]
Great!

**Pull request merged**